docs/3.x.x/installation-options/quick-start-detailed.md

```diff
@@ -0,0 +1,400 @@
+
+   - [List entries (GET)](#list-entries-get)
+   - [Get a specific entry (GET)](#get-a-specific-entry-get)
+   - [Create data (POST)](#create-data-post)
+   - [Update data (PUT)](#update-data-put)
+   - [Delete data (DELETE)](#delete-data-delete)
+
+---
+
+## 1. Create a project
+
+Navigate to your parent "projects" directory in your command line.
+
+```bash
+str@pi:~/Desktop/Strapi/Projects$
+```
+
+Type the following in your command line and press enter to create a Strapi Quick Start project.
+
+```bash
+str@pi:~/Desktop/Strapi/Projects$ strapi new my-project --quickstart
+```
+
+::: warning NOTE
+You can replace the "my-project" project name with any name you want.
+
+:::
+
+::: warning NOTE
+
+When you create a new Quick Start project in Strapi, you will download all the node modules, the Strapi files necessary and the Strapi plugin files. THIS STEP CAN TAKE SEVERAL MINUTES DEPENDING ON YOUR INTERNET CONNECTION SPEED. Please wait for the process to complete before cancelling or trying to continue.
+
+:::
+
+::: warning NOTE
+You can replace the "my-project" project name with any name you want.
+
+Quick start will install Strapi using a SQLite database. ([SQLite](https://www.sqlite.org/index.html) is likely used more than all other database engines combined.)
+
+:::
+
+You will see something like this:
+
+```bash
+str@pi:~/Desktop/Strapi/Projects$ strapi new my-project --quickstart
+🚀 Start creating your Strapi application. It might take a minute, please take a coffee ☕️
+
+
+⏳ Testing database connection...
+The app has been connected to the database successfully!
+
+🏗  Application generation:
+✔ Copy dashboard
+✔ Install plugin settings-manager.
+✔ Install plugin content-type-builder.
+✔ Install plugin content-manager.
+✔ Install plugin users-permissions.
+✔ Install plugin email.
+✔ Install plugin upload.
+✔ Link strapi dependency to the project.
+
+👌 Your new application my-project is ready at /home/str/Desktop/Strapi/Projects/my-project.
+
+⚡️ Change directory:
+$ cd my-project
+
+⚡️ Start application:
+$ strapi start
+```
+
+The command will automatically create a Strapi app folder within your parent "projects" directory.
+
+```bash
+str@pi:~/Desktop/Strapi/Projects$ ls
+my-project
+```
+
+This will open your default browser to the Welcome page for creating an admin user.
+
+![Strapi Welcome Page for creating admin user](../assets/quick-start-detailed/strapi-welcome-screen.jpg 'Strapi Welcome Page for creating admin user')
+
+---
+
+## 2. Create an admin user
+
+The first user you create is the root user for your project. This user has all privileges and access rights. You will need to complete the following fields:
+
+1. Username, create a username for login access to your project, eg. paulbocuse
+2. Password, create a unique password for your project
+3. Email Address, this will be used for recovery
+4. Check receive news, this is optional but recommended
+5. Click the "Ready to Start" button
```

Comment:
I would remove the list

Reply:
We kept the list as it works well with the flow.